kolibri/core/assets/src/core-actions.js

```diff
@@ -62,13 +62,15 @@ function _contentSummaryModel(store) {
 function _contentSessionModel(store) {
   const sessionLog = store.state.core.logging.session;
   const mapping = {
-    user: store.state.core.session.user_id,
     start_timestamp: sessionLog.start_timestamp,
     end_timestamp: sessionLog.end_timestamp,
     time_spent: sessionLog.time_spent,
     progress: sessionLog.progress,
     extra_fields: sessionLog.extra_fields,
   };
+  if (store.state.core.session.kind[0] !== UserKinds.SUPERUSER) {
+    mapping.user = store.state.core.session.user_id;
+  }
```

Comment:
I'm not too familiar with the content logging - but would we need similar logic in `_contentSummaryModel`?

@jayoshih I'd like to get your eyes on these changes. The goal was to have device owners log their interactions as anonymous users. Do these changes look correct to you?